shared/utils/urls.ts

```diff
@@ -67,7 +67,11 @@ export function isUrl(text: string) {
  * @returns True if the url is external, false otherwise.
  */
 export function isExternalUrl(url: string) {
-  return !isInternalUrl(url);
+  if (url) {
+    return !isInternalUrl(url);
+  }
+
+  return false;
```

Comment:
[short-circuit eval](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#short-circuit_evaluation)?
```suggestion
  return !!url && !isInternalUrl(url);
```

Reply:
Nice catch